lib/index.spec.js

```diff
@@ -513,7 +513,7 @@ describe('request error handler', function () {
             'application/json': {
               schema: JSON.stringify({
                 type: 'string',
-                enum: ['a']
```

Comment:
Let's have two separate test cases for enum: one with `['a']` and one with `['a', 'c']`.

Reply:
Done